EXAMPLES.txt

```diff
@@ -9,24 +9,27 @@ Usage Examples
 
 Fetch Google's home page:
 
-    ./http_client -H www.google.com -s 74.125.22.106:443 -p /
+    ./http_client -H www.google.com -s 443 -p /
 
     or even
```

Comment:
The "or even" part was referring to the IPv6 addresses specified as the value to the `-s` option.  Without the IPv6 address, this should probably be rephrased or, likely even better, the second example can be dropped.